src/application/app.rs

```diff
@@ -4,13 +4,21 @@ use crate::{application::State, backend::db};
 pub enum InputMode {
     Command,
     AddTask,
+    Prompt,
+}
+
+#[derive(PartialEq)]
+pub enum Actions {
+    AddTask,
```

Comment:
If AddTask is not used anywhere i would get rid of this Action

Reply:
I know it is used in during initialization of App by beside that it doesn't have any usage